bot.py

```diff
@@ -334,6 +354,35 @@ async def listen_for_response(
 discord.utils.setup_logging(handler=handler)
 
 
+@bot.tree.command(
+    description="Syncs command list. Any new commands will be available for use.",
+)
+@app_commands.checks.has_any_role(ROLE_STAFF, ROLE_VIP)
+async def sync(interaction: discord.Interaction):
```

Comment:
Instead of just having a new file that you have to run to sync the `/sync` command, just make this a normal command and not an app command. Like `!sync`. Then you can get rid of the other file.

Reply:
There is a check to avoid legacy commands in `on_message`, so it would require modifying the regex, looking at the argument, then passing it to be processed iff it begins with `"sync "`. Idk, it just seems a bit messy for something super basic + there being two separate definitions of where "sync" is referenced (one in the command function signature, one in `on_message()`).
 
If we still go this route, would we want to opt for a [hybrid command](https://discordpy.readthedocs.io/en/latest/ext/commands/api.html?highlight=commands%20bot#discord.ext.commands.hybrid_command) instead?